OSDOCS-11609 NodePort Service noted in considerations (#81819)
openshift/openshift-docs: modules/nw-aws-load-balancer-operator-considerations.adoc

* The IP traffic mode only works on AWS Elastic Kubernetes Service (EKS). The AWS Load Balancer Operator disables the IP traffic mode for the AWS Load Balancer Controller. As a result of disabling the IP traffic mode, the AWS Load Balancer Controller cannot use the pod readiness gate.

* The AWS Load Balancer Operator adds command-line flags such as `--disable-ingress-class-annotation` and `--disable-ingress-group-name-annotation` to the AWS Load Balancer Controller. Therefore, the AWS Load Balancer Operator does not allow using the `kubernetes.io/ingress.class` and `alb.ingress.kubernetes.io/group.name` annotations in the `Ingress` resource.

> * The AWS Load Balancer Operator must be configured so the SVC is "NodePort" type instead of "ClusterIP" type.

Reviewer: Please replace "ClusterIP" with "LoadBalancer", so it can be:
`The AWS Load Balancer Operator must be configured so the SVC is "NodePort" type instead of "LoadBalancer" type.`

Author: Of course, yes. Looks like this might need to be included in the install documentation as well?

Reviewer 2: Thanks for raising this PR, Jessi. No installation doc update should be needed as the Networking docs don't really connect to one one platform.

Author: @dfitzmau My apologies, I will need to add this as a consideration in the AWS Load Balancer install or post-install configuration documentation.

> * The AWS Load Balancer Operator must be configured so the SVC is `NodePort` type instead of `LoadBalancer` type.

Reviewer 3: ```suggestion
* You have configured the AWS Load Balancer Operator so that the SVC type is `NodePort` and not `LoadBalancer`.
```

Reviewer 3: Expand first occurence of AWS and SVC?

Reviewer 3: https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites

Reviewer: Since the service types are ClusterIP, NodePort, and LoadBalancer, so maybe it is better to add the ClusterIP, so it could be the following, thanks.
`You have configured the AWS Load Balancer Operator so that the SVC type is `NodePort` and not `LoadBalancer` or `ClusterIP``

Author: Absolutely

Author: Done.
